apps/pmm-ui/src/components/HomePage/HomePage.tsx

```diff
@@ -0,0 +1,110 @@
+import React, { FC } from 'react';
+import { CustomScrollbar, Button, useStyles2 } from '@grafana/ui';
+import { GrafanaTheme2 } from '@grafana/data';
+import { css } from '@emotion/css';
+import { ActionCard } from './comonents/ActionCard';
+import imgDbHealth from './assets/db-health.svg';
+import imgDiscoverPatterns from './assets/discover-patterns.svg';
+import imgOptimizeAndImprove from './assets/optimize-and-improve.svg';
+import imgMaintainPerformance from './assets/maintain-performance.svg';
+import { ActionContainer } from './comonents/ActionContainer';
+import { HomePageTipsContainer } from './comonents/HomePageTipsContainer';
+import { Messages } from './HomePage.messages';
+
+interface HomePageProps {
+  onHelpCenterButtonClick?: () => void;
+  userId: number;
+}
+
+export const HomePage: FC<HomePageProps> = (props) => {
+  const styles = useStyles2(getStyles);
+
+  const { onHelpCenterButtonClick, userId } = props;
+  return (
+    <CustomScrollbar>
+      <div className={styles.welcomePage}>
+        <div className={styles.welcomePageContent}>
+          <ActionContainer
+            headerType="h1"
+            header={Messages.welcomeHeader}
+            text={Messages.welcomeText}
+            actionsSpacing="none"
+            actionsJustify="center"
+          >
+            <div className={styles.actionCardPair}>
+              <ActionCard text="Monitor database health" imgSrc={imgDbHealth} imgAlt="Monitor database health" />
```

Comment:
I think it would make sense to have imgAlts in the messages file.

Also there are some missings texts as "Open Help Center"
 etc.